### PR TITLE
open-cluster-management/backlog#7313 --In review

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by jnpacker — APPROVED** [review]

File: manage_applications/manage_apps_git.adoc [hunk 1]
```diff
@@ -60,125 +60,62 @@ If you enter a new Ansible Tower secret name to create a new secret, you need to
...
+===  GitOps example
 
-[#sample-repo-git-ops]
-=== Sample Repo for GitOps
+Folders in this example are defined and named, with each folder containing applications or configurations that are run on managed clusters:
 
-In the following example repository, you see a folder for each hub cluster. You can also create a repository for each hub cluster, or a branch for each hub cluster. 
+ - Root folder `managed-subscriptions`: Contains subscriptions that target the `common-managed` folder.
+ - Subfolder `apps/`: Used to subscribe applications in `common-managed` folder with placement to `managed-clusters`.
+ - Subfolder `config/`: Used to subscribe configurations in `common-managed` with placement to `managed-clusters`.
```
Comment (dockerymick) [inline manage_applications/manage_apps_git.adoc]: should this be `common-managed` folder?

File: manage_applications/manage_apps_git.adoc [hunk 2]
```diff
@@ -60,125 +60,62 @@ If you enter a new Ansible Tower secret name to create a new secret, you need to
...
-----
+. Subscriptions and policies are downloaded and applied to the hub cluster from the `managed-subscription/` folder.
 
-[#git-ops-hub-clusters]
-=== GitOps application to hub clusters
+ - The subscriptions and policies in the `managed-subscription/` folder then perform work on the managed clusters based on the placement. 
+ - Placement determines which `managed-clusters` each subscription or policy affects.  
+ - The subscriptions or policies define what is on the clusters that match their placement.
+ 
+. Subscriptions apply content from the `common-managed/` folder to `managed-clusters` that match the placement rules. This also applies common applications and confugirations to all `managed-clusters` that match the placement rules.
```
Comment (dockerymick) [inline manage_applications/manage_apps_git.adoc]: Is the backslash required for the folder name. On line 74 and 75 there is no backslash

File: manage_applications/manage_apps_git.adoc [hunk 3]
```diff
@@ -60,125 +60,62 @@ If you enter a new Ansible Tower secret name to create a new secret, you need to
...
-    operators.yaml
-----
+* An example of the `root-subscripition`: https://github.com/open-cluster-management/application-samples/tree/main/subscriptions/subscribe-all[application-subscribe-all].
 
-[#apply-gitops]
-=== Apply GitOps
+* Examples of subscriptions that point to other folders in the same repository: https://github.com/open-cluster-management/application-samples/tree/main/subscriptions[subscribe-all].
 
-With the previous sample combination, you can specify the following:
+* An example of the `common-managed` folder with application artifacts:vhttps://github.com/open-cluster-management/application-samples/tree/main/subscriptions/nginx[nginx-apps].
```
Comment (dockerymick) [inline manage_applications/manage_apps_git.adoc]: remove the `v` in https

File: manage_applications/manage_apps_git.adoc [hunk 4]
```diff
@@ -60,125 +60,62 @@ If you enter a new Ansible Tower secret name to create a new secret, you need to
...
-managed-cluster-common/
-  policies/
-    policy-0.yaml
-    policy-1.yaml
-  hub-policies/
-    policy-0.yaml
-    vault.yaml
-    operators.yaml
-----
+* An example of the `root-subscripition`: https://github.com/open-cluster-management/application-samples/tree/main/subscriptions/subscribe-all[application-subscribe-all].
```
Comment (dockerymick) [inline manage_applications/manage_apps_git.adoc]: See application-subscribe-all for an example of the `root-subscription`

File: manage_applications/manage_apps_git.adoc (hunk 3 above)
Comment (dockerymick) [reply manage_applications/manage_apps_git.adoc]: I would say "See the `common-managed` folder for an example with application artifacts"

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: manage_applications/manage_apps_git.adoc (hunk 1 above)
Comment (swopebe) [reply manage_applications/manage_apps_git.adoc]: yes!

File: manage_applications/manage_apps_git.adoc (hunk 2 above)
Comment (swopebe) [reply manage_applications/manage_apps_git.adoc]: maybe not? I will examine.

File: manage_applications/manage_apps_git.adoc (hunk 3 above)
Comment (swopebe) [reply manage_applications/manage_apps_git.adoc]: I just kept these as a list with colon.

File: manage_applications/manage_apps_git.adoc (hunk 2 above)
Comment (swopebe) [reply manage_applications/manage_apps_git.adoc]: keeping it for `root-subscription/` removing it from others.

**Review by jnpacker — APPROVED** [review]

File: manage_applications/manage_apps_git.adoc (hunk 4 above)
Comment (swopebe) [reply manage_applications/manage_apps_git.adoc]: changed all those

File: manage_applications/manage_apps_git.adoc [hunk 5]
```diff
@@ -60,125 +60,62 @@ If you enter a new Ansible Tower secret name to create a new secret, you need to
...
-managed-cluster-common/
-  policies/
-    policy-0.yaml
-    policy-1.yaml
-  hub-policies/
-    policy-0.yaml
-    vault.yaml
-    operators.yaml
-----
+* For an example of `root-subscription`, see https://github.com/open-cluster-management/application-samples/tree/main/subscriptions/subscribe-all[application-subscribe-all].
```
Comment (dockerymick) [inline manage_applications/manage_apps_git.adoc]: `root-subscription` to `root-subscription/`

File: manage_applications/manage_apps_git.adoc [hunk 6]
```diff
@@ -60,125 +60,62 @@ If you enter a new Ansible Tower secret name to create a new secret, you need to
...
 
-With the previous sample combination, you can specify the following:
+* See an example of the `common-managed` folder with application artifacts in the https://github.com/open-cluster-management/application-samples/tree/main/subscriptions/nginx[nginx-apps] repository.
 
-. A root subscription that can be applied with a CLI command. The root subscription will subscribe back to this repository to apply all the YAML to the hub cluster.
-. The subscription from step 1, which applies application and configuration subscriptions from `common-managed/`.
-. The configuration subscription in step 2, which applies the resources defined in `managed-cluster-common/`.
-. The policies that are defined in `managed-cluster-common/` are also applied to the hub cluster by the subscription in step 1. These policies include those targeted toward the hub cluster, as well as those that target managed clusters.
+* See policy examples in https://github.com/open-cluster-management/policy-collection[Policy collection].
```
Comment (dockerymick) [inline manage_applications/manage_apps_git.adoc:121]: Policy collection to policy-collection

**Review by dockerymick — APPROVED** [review]
2 comments, otherwise this looks good

File: manage_applications/manage_apps_git.adoc (hunk 6 above)
Comment (swopebe) [reply manage_applications/manage_apps_git.adoc:121]: We can name the URL after the URL or the title of the web page. Decided to go after the title of the page on this one.

Comment (swopebe) [reply manage_applications/manage_apps_git.adoc:121]: could also say in the `policy-collection repository`, but both are OK. Link names we have a little freedom, depending on what we want to say.

**Pull request merged**